apps/frontend/src/pages/home/components/starting-page/starting-page.tsx

```diff
@@ -0,0 +1,68 @@
+import {
+	cat,
+	greenFlower,
+	pinkStars,
+	yellowStars,
+	yellowTwinkles,
+} from "~/assets/img/home/home.img.js";
+import { Link } from "~/libs/components/components.js";
+import { AppRoute } from "~/libs/enums/enums.js";
+import { getClassNames } from "~/libs/helpers/get-class-names.js";
+
+import styles from "./styles.module.css";
+
+const StartingPage: React.FC = () => (
```

Comment:
rename to Hero (HeroSection ...)
```suggestion
const Hero: React.FC = () => (
```